src/env.ts

```diff
@@ -9,23 +9,22 @@
 import * as lambda from "@aws-cdk/aws-lambda";
 import log from "loglevel";
 
-export const enableDDTracingEnvVar = "DD_TRACE_ENABLED";
-export const injectLogContextEnvVar = "DD_LOGS_INJECTION";
-
-export const defaultEnvVar = {
```

Comment:
I moved this to `datadog.ts` and renamed it to `defaultProps` because these aren't environment variables. Some of them set environment variables, but `addLayers` for example has nothing to do with environment variables.

Reply:
👍